torch/testing/_internal/common_device_type.py

```diff
@@ -482,6 +483,13 @@ def instantiate_device_type_tests(generic_test_class, scope, except_for=None, on
         scope[class_name] = device_type_test_class
 
 
+# Category of dtypes to run an OpInfo-based test for
+class OpDTypes(Enum):
```

Comment:
Elaborate here on on how a test writer should choose between these options. We want to be able to direct developers less familiar with PyTorch's test suite to this comment and have them be able to choose the correct option.